bower.json

```diff
@@ -2,6 +2,7 @@
     "name": "adapt-devtools",
     "repository": "git://github.com/cgkineo/adapt-devtools",
     "framework": ">=2.2.0",
+    "version": "2.1.19",
```

Comment:
@moloko unless GH is deceiving me I'm seeing two versions and we've got back a version...

Reply:
@moloko ok now it looks fine - that was really weird, looks fine now

Reply:
TBH I just fixed the conflict without worrying too much about what it should be; I'll check it all once I've merged this and the other one!